### ISSUE TITLE
Status Update as of 5/5/2016

### ISSUE THREAD

**Issue by sahat** [sahat/satellizer#816]
First off, I should apologize for the delay in responding to the GitHub issues. A lot has been happening in my personal life lately. Not to mention I am simultaneously working on several other open-source projects, while trying to start a consulting business with a friend. Also most of my coding hours have been going into https://github.com/sahat/boilerplate (name will likely change)  which is scheduled to release this month.

Now, as far as my short-term plans for Satellizer:

1. Create a dedicated _GitHub Pages_ site with docs.
2. Modularize and convert existing code to ES6. ([it is mostly done on a separate branch](https://github.com/sahat/satellizer/tree/typescript/src) but haven't had a chance to test it)
3. Create **react-satellizer** port as a standalone package or as part of this project, which could contain several wrappers: Angular 1, Angular 2, React. The **ui.router** does something similar. It really depends on how much of "core" I could re-use between the frameworks. Simplified version of Satellizer for React + Redux already [exists here](https://github.com/sahat/boilerplate/blob/docs/generators/js-framework/modules/react/actions/oauth.js).
4. Add Angular 2 support after Step 2 is completed.

Issues would need to be triaged as well, but that's a separate task. I would really love some help here.  You will be given a contributor access to close issues and manage labels. Speaking of GitHub issues, I would also appreciate if someone could compile a list of closed issues that should be in the documentation because a question, an answer or a discussion that followed was really helpful.

Thanks for your patience and your continued use of Satellizer!
